app/src/components/common/BalanceRow.vue

```diff
@@ -4,9 +4,12 @@
       <img
         class="currency-flag"
         :src="
-          currentNetwork.coinLookup.find(({ viewDenom }) => balance.denom)
-            ? currentNetwork.coinLookup.find(({ viewDenom }) => balance.denom)
-                .icon
+          currentNetwork.coinLookup.find(
```

Comment:
what is the fix? can't spot the difference

Reply:
`currentNetwork.coinLookup.find(({ viewDenom }) => balance.denom`
vs.
`currentNetwork.coinLookup.find(({ viewDenom }) => viewDenom === balance.denom`

Reply:
but yeah i can't tell what actually changes either

Reply:
I'm surprised you haven't merged this yet :sweat_smile: 

The difference is that the first element in the Array with a truthy viewDenom (any value) will be taken as the icon. So, for example, in Terra testnet the fist one is KRT, so all tokens will have the KRT icon. In Terra mainnet it will be the Luna icon, and so on